packages/cli/README.md

```diff
@@ -8,13 +8,19 @@
 
 ---
 
+|                              📊 Getting Started                              |                           🌐 Portal Integration                            |                          🛠️ CI Automation                          |
+| :--------------------------------------------------------------------------: | :------------------------------------------------------------------------: | :----------------------------------------------------------------: |
+| **[How to setup](./packages/cli/README.md#getting-started)** a basic project | Sort, filter **[your goals](./packages/cli/README.md#portal-integration)** | Updates **[on every PR](./packages/cli/README.md#ci-integration)** |
```

Comment:
This README is rendered on npmjs.com as well: https://www.npmjs.com/package/@code-pushup/cli

Relative file links won't work there. In this case, it's an easy fix, because the referenced sections are in the same document. (In other cases, the full GitHub URL has to be used.)

```suggestion
| **[How to setup](#getting-started)** a basic project | Sort, filter **[your goals](#portal-integration)** | Updates **[on every PR](#ci-integration)** |
```

(Relative links are fine in files other than package READMEs, e.g. the root `README.md`).

Reply:
done